sdk/openai/Azure.OpenAI.Inference/api/Azure.OpenAI.Inference.netstandard2.0.cs

```diff
@@ -0,0 +1,98 @@
+        public OpenAIClient(System.Uri endpoint, Azure.AzureKeyCredential credential) { }
+        public OpenAIClient(System.Uri endpoint, Azure.AzureKeyCredential credential, Azure.OpenAI.Inference.OpenAIClientOptions options) { }
+        public OpenAIClient(System.Uri endpoint, Azure.Core.TokenCredential credential) { }
+        public OpenAIClient(System.Uri endpoint, Azure.Core.TokenCredential credential, Azure.OpenAI.Inference.OpenAIClientOptions options) { }
+        public virtual Azure.Core.Pipeline.HttpPipeline Pipeline { get { throw null; } }
+        public virtual Azure.Response Completions(string deploymentId, Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
+        public virtual Azure.Response<Azure.OpenAI.Inference.Models.Completion> Completions(string deploymentId, Azure.OpenAI.Inference.Models.CompletionsRequest completionsRequest, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response> CompletionsAsync(string deploymentId, Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.OpenAI.Inference.Models.Completion>> CompletionsAsync(string deploymentId, Azure.OpenAI.Inference.Models.CompletionsRequest completionsRequest, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response Embeddings(string deploymentId, Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
+        public virtual Azure.Response<Azure.OpenAI.Inference.Models.Embeddings> Embeddings(string deploymentId, Azure.OpenAI.Inference.Models.EmbeddingsRequest embeddingsRequest, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response> EmbeddingsAsync(string deploymentId, Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.OpenAI.Inference.Models.Embeddings>> EmbeddingsAsync(string deploymentId, Azure.OpenAI.Inference.Models.EmbeddingsRequest embeddingsRequest, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+    }
+    public partial class OpenAIClientOptions : Azure.Core.ClientOptions
+    {
+        public OpenAIClientOptions(Azure.OpenAI.Inference.OpenAIClientOptions.ServiceVersion version = Azure.OpenAI.Inference.OpenAIClientOptions.ServiceVersion.V2022_06_01_Preview) { }
+        public enum ServiceVersion
+        {
+            V2022_06_01_Preview = 1,
+        }
+    }
+}
+namespace Azure.OpenAI.Inference.Models
+{
+    public partial class Choice
```

Comment:
I'm guessing we will want to rename all of these types to be multiple words?

Reply:
yeah this is garbage in garbage out.  the archboard review will catch all the cadl spec changes needed